frontend/components/pod_components/PartnerPod.js

```diff
@@ -36,7 +36,7 @@ export default function PartnerPodScreen({ navigation }) {
         
             <TouchableOpacity 
                 style={styles.block} 
-                onPress={() => navigation.navigate('Random Screen')}
+                onPress={() => navigation.navigate('Outcomes')}
```

Comment:
These lines in the partner pod cause the app to error. I think you need to pass in the information to this pod the same way you did for associates and trainee